CHANGELOG.md

```diff
@@ -44,16 +44,16 @@
 
 - Add icon for syncService
 - Add status for cluster
-- No cluster add guide information
+- Add guide information when there is no cluster
 
 ### Bug Fixes
 
 - Fixed the ssh trust problem when installing application for the first time
 - Fixed the run command
-- Fixed stateful workload get config and state of ⚙️
+- Fixed stateful workload get config and state of ⚙️ icon
```

Comment:
Fixed failing to get config of statefulset 
Fixed status icon of statefulset displaying incorrectly